package.json

```diff
@@ -52,8 +51,7 @@
         "prettier": "^3.0.1",
         "typescript": "^5.1.6",
         "uuid": "^9.0.0",
-        "wdio-chromedriver-service": "^8.1.1",
-        "webdriverio": "8.14.3"
+        "webdriverio": "^8.14.3"
```

Comment:
Update the yarn lock as 8.15.0 is out and it fixes the max instance issue. https://github.com/webdriverio/webdriverio/pull/10952
See if it is still flaky after upgrading.

Reply:
Still very flaky on two instances after this update, had two runs resulting in multiple suites failing on multiple tests, reverted it back to one and it pass all tests on first try..

Reply:
Have we looked at why it might be happending?.

Reply:
I did few more test runs with 2 instances and is similar kind of flakiness today as 1 instance so we could possible keep it as 2?

Reply:
Sorry I'm not following, are you saying 1 is flaky as well?. Have we identified why it is flaky?.

Reply:
They are both flaky but, from checking the logs, when using one instance we get same type and amount of errors and failed builds as on main branch (same errors/behaviour as on main), when we increase the number of instances we get failed tests but no logs indicating what the problem is, just failed tests.

Reply:
Right okay, on 1 it is no flakier than main but on 2, it is error for other reasons.
So we should try to understand why 2 is flaky and if we can't get to the bottom of it, proceed with 1, and add a TODO comment to revisit and up to 2.

Reply:
Seems like 1 is slightly less flaky than main but hard to capture and measure this. That TODO you mention is already added, I will try to get some meaningful error logs from 2.

Reply:
Let's get this moving, as it blocks local development. Can you raise a card to look into this bug?. We can tackle it separately.

Reply:
OK, will merge it and raise the card, thanks.